### ISSUE TITLE
ManifestInvalidRevisionError

### ISSUE THREAD

**Issue by devel0perandr0id420** [RaspberryPiFan/device_brcm_rpi4#13]
$ repo init -u https://android.googlesource.com/platform/manifest -b android-10.0.0_r2
 $ git clone https://github.com/RaspberryPiFan/local_manifests .repo/local_manifests -b android-10
 $ repo sync
On giving the above commands ended up in the following exception 

Checking out projects:   2% (15/731) device/amlogic/yukawa-kernelerror: Cannot checkout device_brcm_rpi4: ManifestInvalidRevisionError: revision android-10 in device_brcm_rpi4 not found
error: in `sync`: revision android-10 in device_brcm_rpi4 not found

Any idea ?

**Comment by devel0perandr0id420** [comment]
Removed .repo/localmaniefest/ contents and gave repo sync again, Now it fetching some projects.I have no idea whats happening :D

**Comment by RaspberryPiFan** [comment]
that's not good.

**Comment by RaspberryPiFan** [comment]
You don't get the pi 4 support

**Comment by RaspberryPiFan** [comment]
Just try again

**Comment by devel0perandr0id420** [comment]
I will do clean build again and update you
.Between were you able to make it up in rpi4?

**Comment by RaspberryPiFan** [comment]
No, but I'm working on it with @ptruter. He got a working kernel. When you're finished with compiling, I'll upload the new kernel

**Comment by devel0perandr0id420** [comment]
I have restarted the activity again and  repo sync is in progress. Fingers crossed!

**Comment by devel0perandr0id420** [comment]
Checking out projects:   2% (15/731) device/amlogic/yukawa-kernelerror: Cannot checkout device_brcm_rpi4: ManifestInvalidRevisionError: revision android-10 in device_brcm_rpi4 not found
error: in `sync`: revision android-10 in device_brcm_rpi4 not found

Still getting the same exception.

**Comment by vitek499** [comment]
resolution of this problem is here: https://github.com/RaspberryPiFan/local_manifests/pull/4
